python/pc_ble_driver_py/ble_adapter.py

```diff
@@ -289,12 +300,41 @@ def write_cmd(self, conn_handle, uuid, data):
         self.evt_sync[conn_handle].wait(evt = BLEEvtID.evt_tx_complete)
 
 
+    @NordicSemiErrorCheck(expected = BLEGapSecStatus.success)
+    def pair(self, conn_handle):
```

Comment:
I'd rather you called this authenticate() that way we cover better pairing and bonding